pages/pipelines/notifications.md.erb

```diff
@@ -0,0 +1,58 @@
+# Triggering Notifications
+
+The `notify` attribute allows you to trigger build notifications to different services. You can also choose to conditionally send notifications based on pipeline events like build status.
+
+<%= toc %>
+
+Add notifications to your pipeline with the `notify` attribute. This sits  at the same level as `steps` in your `pipeline.yml` file.
+
+The below example will send a notification email every time a build is created on this pipeline: 
+
+```yaml
+steps:
+  - command: "tests.sh"
+  - wait
+  - command: "build.sh"
+
+notify:
+  - email: "dev@acmeinc.com"
+```
+
+You can trigger notifications as emails or as a message in a Basecamp Campfire.
+
+## Conditional notifications
+
+You can choose to only trigger notifications when a statement evaluates to true using the `if` attribute.
+
+In the below example the email notification will only be triggered if the build passes:
+
+```yaml
+notify:
+  - email: "dev@acmeinc.com"
+    if: "build.status == passed"
+```
+
+## Email
+
+Add an email notification to your pipeline using the `email` attribute of the `notify` yaml block:
```

Comment:
You can add multiple emails by duplicating the email step:

```
- email: "hi@hi.com"
- email: "hi@frogs.com"
```
I'd really like this to be a nicer experience but for now it might be worth adding a note that it  is possible  to notify multiple emails?

Reply:
Oh nice! Yep definitely adding that.